.coveragerc

```diff
@@ -12,6 +12,13 @@ omit =
     *urllib3/_backends/twisted_backend.py
     *urllib3/contrib/socks.py
 
+[paths]
+source =
+    src/urllib3
+    .tox/*/lib/python*/site-packages/urllib3
+    .tox\*\Lib\site-packages\urllib3
+    .tox/pypy*/site-packages/urllib3
```

Comment:
We don't need those lines, we use a different approach due to bleaching (see above)